closed/adds/jdk/src/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -179,6 +184,22 @@ public static boolean isEnabled() {
         return securityEnabled;
     }
 
+    /**
+     * Get restricted security mode secure random provider.
+     *
+     * Restricted security mode secure random provider can only
+     * be called in restricted security mode.
+     *
+     * @return the secure random provider
+     */
+    public static String getRandomProvider() {
+        if (!securityEnabled) {
+            printStackTraceAndExit(
+                    "Restricted security mode secure random provider can only be used when restricted security mode is enabled.");
+        }
+        return restricts.jdkSecureRandomProvider;
+    }
```

Comment:
This method should be removed - it's not used.
The field `jdkSecureRandomProvider` and all references  should likewise be removed.

Reply:
Removed.